modules/ROOT/pages/tools/neo4j-admin/validate-config.adoc

```diff
@@ -0,0 +1,62 @@
+:description: How to validate configurations using Neo4j Admin.
+[[neo4j-admin-validate-config]]
+= Validate configurations
+
+The `neo4j-admin server validate-config` command validates the Neo4j and Log4j configurations.
+It is located in the `bin` directory and is used to validate the configuration files before starting Neo4j.
+The command is available in Neo4j v5.5 and later.
```

Comment:
Are we mentioning the minor versions now? Because when 5 was released there was that discussion that we should only refer to Neo4j 5, no matter what minor version

Reply:
Yes, we mention the minor version when a new feature is introduced or updated. Otherwise, the users will wonder why they don't have the feature even though they are on version 5.